### PR TITLE
Pixel polish for ERC20 Desktop

### PR DESCRIPTION
**Pull request by juan-cortes**
### Things done

- [x] Made the breadcrumbs follow design
- [x] Fixed styles for the range selector
- [x] Modified the expand/collapse behaviour of token rows for a full row cta
- [x] Asset distribution amount color change

- [x] Add tooltip for Grid Mode  Token Account cards to show the parent account name

image
image
image

### Type

UI Polish

### CODE REVIEW

**Comment by juan-cortes** [comment]
@dasilvarosa can you check wording on the screenshots please?

File: src/components/AccountsPage/AccountGridItem/Header.js [hunk 1]
```diff
@@ -56,10 +56,7 @@ class Header extends PureComponent<{
     return (
       <Box flow={4}>
         <Box horizontal ff="Open Sans|SemiBold" flow={3} alignItems="center">
-          <ParentCryptoCurrencyIcon
-            currency={currency}
-            parentCurrency={parentAccount && parentAccount.currency}
-          />
+          <ParentCryptoCurrencyIcon currency={currency} parent={parentAccount} />
```
Comment (gre) [inline src/components/AccountsPage/AccountGridItem/Header.js]: parent does not exist as a prop in ParentCryptoCurrencyIcon 🤔

File: src/components/AccountsPage/AccountList/Header.js [hunk 2]
```diff
@@ -72,9 +71,9 @@ class Header extends PureComponent<Props, { focused: boolean }> {
           onChange={onTextChange}
           value={search}
         />
-        <Text color="dark" ff="Museo Sans" fontSize={6} data-e2e="dashboard_AccountCount">
+        <Box>
```
Comment (gre) [inline src/components/AccountsPage/AccountList/Header.js]: is this wrapping required or can Box be dropped?

File: src/components/AccountsPage/AccountGridItem/Header.js (hunk 1 above)
Comment (gre) [reply src/components/AccountsPage/AccountGridItem/Header.js]: nevermind, i didn't see the related change.

are you sure there are no other reference of that component?

**Comment by gre** [comment]
@juan-cortes if you can see my lil feedback. i think we can then merge.

File: src/components/AccountsPage/AccountGridItem/Header.js (hunk 1 above)
Comment (juan-cortes) [reply src/components/AccountsPage/AccountGridItem/Header.js]: I am

File: src/components/AccountsPage/AccountList/Header.js (hunk 2 above)
Comment (juan-cortes) [reply src/components/AccountsPage/AccountList/Header.js]: indeed, no need for box here

**Comment by juan-cortes** [comment]
@gre  ready from my side

**Pull request merged**